easybuild/easyconfigs/b/binutils/binutils-2.25-GCCcore-4.9.3-binutils-2.25.eb

```diff
@@ -6,7 +6,7 @@ version = '2.25'
 homepage = 'http://directory.fsf.org/project/binutils/'
 description = "binutils: GNU binary utilities"
 
-toolchain = {'name': 'GCC', 'version': '4.9.3-binutils-%s' % version}
+toolchain = {'name': 'GCCcore', 'version': '4.9.3-binutils-%s' % version}
```

Comment:
existing easyconfigs should not be removed/renamed, instead a new one should be added instead